zgw/zaken-api/src/main/kotlin/com/ritense/zakenapi/client/ZakenApiClient.kt

```diff
@@ -241,6 +245,30 @@ class ZakenApiClient(
         return result
     }
 
+    fun updateZaakRol(
+        authentication: ZakenApiAuthentication,
+        baseUrl: URI,
+        rolUuid: UUID,
+        rol: Rol
+    ): Rol {
+        val result = buildRestClient(authentication)
+            .put()
+            .uri {
+                ClientTools.baseUrlToBuilder(it, baseUrl)
+                    .path("rollen/${rolUuid}")
+                    .build()
+            }
+            .body(rol)
+            .retrieve()
+            .body<Rol>() ?: throw NoSuchElementException("No body was returned when updating rol($rolUuid)")
```

Comment:
This is the wrong exception. Please make a more specific exception for this. We do this all over Valtimo, so there's plenty of examples to pick from